Attempting to fix the TravisCi fail

mbernier opened the pull request:
It looks like the JAVA_HOME is not set properly, trying this change from https://github.com/travis-ci/travis-ci/issues/2616 to see if this works.

Commit:
Attempting to fix the TravisCi fail

It looks like the JAVA_HOME is not set properly, trying this change from https://github.com/travis-ci/travis-ci/issues/2616 to see if this works.

SendGridDX commented:
[![CLA assistant check](https://cla.sendgrid.com/pull/badge/signed)](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=27) <br/>All committers have signed the CLA.

Commit:
Another update, based on Travis docs

https://docs.travis-ci.com/user/languages/java/

It looks like the fact that we have gradle.build and gradlew in the root, we don't need to run the gradle build script.

Also, we can add caching to make things move a little faster ;)

mbernier commented:
Still failing ...and I am out of my depths on this one. Maybe this? https://github.com/codecov/example-gradle/blob/master/gradlew
